src/pages/Registration/Steps/Banking/useSubmit.ts

```diff
@@ -1,43 +1,71 @@
-import { SubmitHandler, UseFormReturn } from "react-hook-form";
+import { useCallback, useState } from "react";
 import { useNavigate } from "react-router-dom";
-import { FormValues as FV } from "./types";
+import { CreateRecipientRequest } from "types/aws";
+import { FileDropzoneAsset } from "types/components";
+import { useCreateRecipientAccountMutation } from "services/aws/bankDetails";
 import { useUpdateRegMutation } from "services/aws/registration";
 import { useErrorContext } from "contexts/ErrorContext";
+import { getFilePreviews } from "helpers";
+import { GENERIC_ERROR_MESSAGE } from "constants/common";
 import { steps } from "../../routes";
 import { useRegState } from "../StepGuard";
 
-export default function useSubmit(form: UseFormReturn<FV>) {
+export default function useSubmit() {
   const { data } = useRegState<5>();
-  const {
-    handleSubmit,
-    formState: { isDirty, isSubmitting },
-  } = form;
-
+  const [createRecipientAccount] = useCreateRecipientAccountMutation();
   const [updateReg] = useUpdateRegMutation();
   const { handleError } = useErrorContext();
   const navigate = useNavigate();
+  const [isSubmitting, setSubmitting] = useState(false);
+
+  const submit = useCallback(
```

Comment:
nit: `useCallback` is [generally not necessary](https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse), as there's no need for `submit` to be referentially equal all the time (unless used in `useMemo` or `useEffect` [deps](https://kentcdodds.com/blog/usememo-and-usecallback#dependencies-lists))

Reply:
You're right, removed.

See https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2488/commits/6394ab9d733a292ae9e90dae55d419ed7c3e6e38